source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -0,0 +1,190 @@
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+          _id: 1
+        result:
+          insertedId: 1
+      - name: abortTransaction
+        object: session0
+    expectations:
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - _id: 1
+            ordered: true
+            readConcern:
+            lsid: session0
+            txnNumber:
+              $numberLong: "1"
+            startTransaction: true
+            autocommit: false
+            writeConcern:
+          command_name: insert
+          database_name: *database_name
+      - command_started_event:
+          command:
+            abortTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: abortTransaction
+          database_name: admin
+      - command_started_event:
+          command:
+            abortTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: abortTransaction
+          database_name: admin
+    outcome:
+      collection:
+        data: []
+
+  - description: abortTransaction does not retry without RetryableWriteError label
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+        failCommands: ["abortTransaction"]
+        errorLabels: []
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+          session: session0
+          document:
+            _id: 1
+          result:
+            insertedId: 1
+      - name: abortTransaction
+        object: session0
+    expectations:
+      - command_started_event:
+        command:
+          insert: *collection_name
+          documents:
+            - _id: 1
+          ordered: true
+          readConcern:
+          lsid: session0
+          txnNumber:
+            $numberLong: "1"
+          startTransaction: true
+          autocommit: false
+          writeConcern:
+        command_name: insert
+        database_name: *database_name
+      - command_started_event:
+          command:
+            abortTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: abortTransaction
+          database_name: admin
+    outcome:
+      collection:
+          data: []
+
+  - description: abortTransaction retries once after network exception
```

Comment:
Is this any different than "abortTransaction succeeds after connection error" in `retryable-abort.yml`? Perhaps we can just add label assertions to existing abort tests, as those shouldn't require a particular server version.